### ISSUE TITLE
ConvertX not redirrecting

### ISSUE THREAD

**Issue by Techsmith404** [ItsEcholot/ContainerNursery#60]
Everything else works beautifully. I have like 6 other containers routed thru nursery and they work perfectly fine. But for some reason when I try to go to convertx it says it's waking up, I can see the container start, but I never get redirected. If I update the config file in any way it'll redirect to the convertx site, but then the next time I go to the site I'm stuck at waking up. I tried setting the method to GET with no luck. Tried enabling https with no luck. The log doesn't show anything other than container starting. What am I missing?

**Comment by ItsEcholot** [comment]
This sounds like the convertx container not responding with a valid status code and thus not being marked as "ready to receive connections" by ContainerNursery.
Try setting the `CN_LOG_LEVEL` environment to `debug` to get more information in the logs.

**Comment by Techsmith404** [comment]
Yeah it's saying status 404 and there are no headers. What can I do to fix this? Also I'm noticing some of my containers are starting and stopping on their own. I don't have anything set to ping them, I took them off of uptime kuma thinking that was the cause but it's still happening. Idk if it's something to do with cloudflare tunnels or maybe its bots or people trying to access my shit idk. Have you seen that before?

**Comment by ItsEcholot** [comment]
404 isn't a positive status and that's the reason that ContainerNursery assumes the container isn't ready.
This seems like an issue with the ConvertX container and there's not much I can do here.

Related to your second issue: No I haven't seen that before but here too the debug logging could help shed some light which request / connection triggers the wakeup of your containers.

**Comment by drgshub** [comment]
Having the same exact issue with nextcloud. Any workaround for this?
